app/models/stories.model.js

```diff
@@ -0,0 +1,24 @@
+module.exports = (sequelize, Sequelize) => {
+    const Story = sequelize.define("story", {
+        storyId: {
+            type: Sequelize.INTEGER,
+            primaryKey: true,
+            autoIncrement: false,
+        },
+        userId: {
+            type: Sequelize.INTEGER,
+            allowNull: false,
+        },
+        story: {
+            type: Sequelize.TEXT,
+            allowNull: false,
+        },
+
+        parentId: {
+            type: Sequelize.STRING,
```

Comment:
shouldn't the parentId also be an INTEGER if the id of a story is an INTEGER?

Reply:
Yes.